### PR TITLE
fix(orc8r): Fix grpc_internal port for smsd service

### PR DESCRIPTION
**Pull request by xbend**
Fixed port number for grpc_internal introduced in #12564.
Port number for grpc_internal had same value as grpc port.
Changed grpc_internal port to grpc port +10 as was done
with other services in PR #12564.

Signed-off-by: Artur Dębski <artur.debski@freedomfi.com>

## Summary

Magma PR https://github.com/magma/magma/pull/12564 got merged despite the fact that Domain Proxy CI jobs indicated a mistake in the helm charts for smsd (https://github.com/magma/magma/runs/6261712285?check_suite_focus=true):
```
Error: INSTALLATION FAILED: Service "orc8r-smsd" is invalid: spec.ports[1]: Duplicate value: core.ServicePort{Name:"", Protocol:"TCP", AppProtocol:(*string)(nil), Port:9180, TargetPort:intstr.IntOrString{Type:0, IntVal:0, StrVal:""}, NodePort:0}
```
This PR fixes the port duplication for smsd service in helm charts.

## Additional Information

- [ ] This change is backwards-breaking

### CODE REVIEW

**Comment by xbend** [comment]
@christinewang5 @hcgatewood @themarwhal @jpad-freedomfi  FYI
Guys, CI showed very clearly that Domain Proxy job was failing due to a mistake in the merged PR https://github.com/magma/magma/pull/12564.
Domain Proxy approvers did not approve the PR, yet it was merged.

We have CI for a reason and neglecting the CI results not only defeats the purpose of the CI in the first place, but also increases the incentive that other code owners my merge their PRs regardless of the CI test results and impact other sub-projects.

**Comment by themarwhal** [comment]
@xbend Is the CI check mentioned in your summary ready be made required? If it is stable and provides important information, it might be good.

**Comment by xbend** [comment]
> @xbend Is the CI check mentioned in your summary ready be made required? If it is stable and provides important information, it might be good.

@themarwhal 
None of the Domain Proxy Ci checks are marked as required. Main reason is that we still see some random flakiness in the different runs for integration checks. Causes are issues known to us and we are working on improvements.

I would personally not qualify the checks as suitable candidates for required checks at this point in time. Soon, but not yet.

Although I do remember required jobs contain flakiness in the past and what we have done during development, is to keep rerunning the failing jobs to get a pass, and report the flake on the slack/board/issue tracker - or consult the owners if the changes could be merged despite the failures.
CI check not being a requirement for a merge does not, or rather, should not just automatically mean one can just press the merge button whenever it lights up green.

**Comment by xbend** [comment]
@arunuke @hcgatewood @ShubhamTatvamasi 
Change is ready for review. We see that codecov gave failing check results but it has nothing to do with the change introduced here, as we only fixed helm.

We would greatly appreciate this be reviewed as the master helm error is blocking all PRs from passing CI tests.

**Review by ShubhamTatvamasi — APPROVED** [review]

**Pull request merged**
